packages/viz/src/BarChart/BarChart.tsx

```diff
@@ -70,6 +70,7 @@ export const HvBarChart = forwardRef<ReactECharts, HvBarChartProps>(
       groupBy,
       splitBy,
       sortBy,
+      filters,
```

Comment:
We need to do the same to the other charts that are extending `HvChartCommonProps`. Otherwise the prop is available but doesn't do nothing.